### ISSUE TITLE
Headhunter - map overlay not visible / overlaps / obscured

### ISSUE THREAD

**Issue by GoogleCodeExporter** [aloone/nulldc#240]
```
The bike sections in Headhunter should have an overhead street map in the top 
right corner - see 
http://www.youtube.com/watch?v=M5ttNYaTUeo&p=8CC817B8BAD234E1&playnext=1&index=2
 from 0:56 onwards.

In nullDC this map is only visible when using D24S8 Mode 2 Z-buffer mode, 
however this causes other problems:

1. The streets on the map overlap the square border of the map by some 
distance. The streets should be contained within the square border.
2. Buildings etc. in the main view overlap/obscure the map. The map should be 
in the foreground at all times.

See attached screenshot, also I've attached a VMU save so you can reproduce 
without going through the lengthy intro.
```

Original issue reported on code.google.com by `matbur...@gmail.com` on 13 Oct 2010 at 10:53

Attachments:
- [Headhunter.jpg](https://storage.googleapis.com/google-code-attachments/nulldc/issue-240/comment-0/Headhunter.jpg)
- [vmu_data_port01.bin](https://storage.googleapis.com/google-code-attachments/nulldc/issue-240/comment-0/vmu_data_port01.bin)

**Comment by GoogleCodeExporter** [comment]
Original comment by `drkIIRaziel` on 16 Oct 2010 at 12:19
- Added labels: **Type-Defect**, **emu-pvr**, **emu-pvr-limits**
- Removed labels: **Type-Enhancement**

**Comment by GoogleCodeExporter** [comment]
```
So I'm searching YouTube for 'Headhunter nulldc' and the first result is one of 
Raz's own videos from Mar 2007, showing an old version of nulldc running 
Headhunter but with the map working correctly, staying within its boundaries:

http://www.youtube.com/results?search_query=nulldc+headhunter&aq=f

So the obvious question is - how was this working almost 4 years ago, but isn't 
now?
```

Original comment by `matbur...@gmail.com` on 22 Jan 2011 at 5:28

**Comment by GoogleCodeExporter** [comment]
```
No comment?
```

Original comment by `matbur...@gmail.com` on 1 Feb 2011 at 11:30

**Comment by GoogleCodeExporter** [comment]
```
i was just about to add this issue, im on disk 2 and still no map overlap, 
tried switching different settings, if i press pause i see it quickly flash in 
top right where it should be but then disappears.
```

Original comment by `cem.a...@gmail.com` on 6 Feb 2011 at 1:33

**Comment by GoogleCodeExporter** [comment]
```
As I mention in the first post, choosing the last z-buffer setting shows the 
map, but with the overlap.

I just wondered why there's a video of nulldc from years back where the map's 
working properly.
```

Original comment by `matbur...@gmail.com` on 6 Feb 2011 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
Because the z-buffer on the dreamcast can store up to 4,294,967,296 depth 
values. These values can be any integer (not just numbers, nans included, 
infinite values included too).

Now, using hardware and/or software and/or drivers that support anything less 
than direct3d 10 would leave space for about 16,777,215 depth values (numbers 
only, from 0 to 1).

Now, the above difference is quite bad, but due to the difference on the way 
the z-buffer works on the dreamcast and a pc video card, there are others, 
worse issues that make errors of this kind unavoidable...

...unless someone makes a direct3d 10 plugin (that will only work for newer 
cards of course). In that case, most issues will be taken care of.

The current approach just works "alright" as a whole. Meaning that it does not 
break most stuff, but it will break others. Of course someone could make some 
piece of software that shows only garbage on nulldc (or other dreamcast 
emulators) on purpose, while it work fine on a real dreamcast.

In short, you'll have to live with this kind of issues for the time being.
```

Original comment by `Mr.PsyMan` on 7 Feb 2011 at 10:22

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the explanation, shame the project's a bit dead now.
```

Original comment by `matbur...@gmail.com` on 7 Feb 2011 at 1:14

**Comment by GoogleCodeExporter** [comment]
```
I remember... Once, someone started DX11Plugin and forgot about it, 
unfortunately.

P.S.
Structurally DX10 and DX11 has no a big difference as far as I know. Maybe we 
need DX11 plugin that may support both: DX10 & DX11?
```

Original comment by `N2008eo...@rambler.ru` on 11 Feb 2011 at 8:55
